chord_metadata_service/mohpackets/tests/models/test_biomarker.py

```diff
@@ -0,0 +1,63 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.test import TestCase
+
+from chord_metadata_service.mohpackets.models import Biomarker, Program
+from chord_metadata_service.mohpackets.permissible_values import HER2_STATUS
+from chord_metadata_service.mohpackets.schemas.model import DonorModelSchema
+from chord_metadata_service.mohpackets.tests.factories import BiomarkerFactory
+from chord_metadata_service.mohpackets.tests.utils import INVALID_ID_LIST
```

Comment:
This and DonorModelSchema is unused -- either the test should use it or the import should be removed

Reply:
yea, I will fix all the linter errors on final